src/lwpCallControl.js

```diff
@@ -316,17 +352,29 @@ export default class extends lwpRenderer {
             </button>
           {{/data.call.held}}
 
-          {{^data.call.muted}}
+          {{^data.call.isAudioMuted}}
             <button id="{{by_id.mute.elementId}}">
               {{i18n.mute}}
             </button>
-          {{/data.call.muted}}
+          {{/data.call.isAudioMuted}}
 
-          {{#data.call.muted}}
+          {{#data.call.isAudioMuted}}
             <button id="{{by_id.unmute.elementId}}">
               {{i18n.unmute}}
             </button>
-          {{/data.call.muted}}
+          {{/data.call.isAudioMuted}}
+
+          {{^data.call.isVideoMuted}}
+             <button id="{{by_id.muteVideo.elementId}}">
```

Comment:
Here is a small format typo.
```suggestion
            <button id="{{by_id.muteVideo.elementId}}">
```

Reply:
Fixed